src/runtime/Module.cpp

```diff
@@ -368,18 +368,27 @@ Instance* Module::instantiate(ExecutionState& state, const ExternVector& imports
 #if !defined(NDEBUG)
 void ModuleFunction::dumpByteCode()
 {
-    printf("function type %p\n", m_functionType);
-    printf("requiredStackSize %u, requiredStackSizeDueToLocal %u\n", m_requiredStackSize, m_requiredStackSizeDueToLocal);
+    printf("%-34s: %-10p\n", "function type", m_functionType);
```

Comment:
The first argument can be part of the string. This looks complicated.

Reply:
I used it to ensure the data will be in one line.

Like this:
```
function type                     : 0x55fff0ee3f80
required stack size               : 32 bytes
required stack size due to local  : 32 bytes
pointer size                      : 8 bytes
bytecode size                     : 160 bytes (pointer-aligned)
```

Otherwise, it would look like this:
```
function type: 0x55fff0ee3f80
required stack size: 32 bytes
required stack size due to local: 32 bytes
pointer size: 8 bytes
bytecode size: 160 bytes (pointer-aligned)
```

Or should I use spaces in the string?